Tests/CacheAdvanceTests/CacheAdvanceTests.swift

```diff
@@ -420,6 +420,62 @@ final class CacheAdvanceTests: XCTestCase {
         }
     }
 
+    func test_performance_append_fillableCache() throws {
+        measure {
+            guard let sut = try? createCache(overwritesOldMessages: false, zeroOutExistingFile: true) else {
```

Comment:
Should we zero out the file before the test starts?

Reply:
Or should we write a new file each time so that doesn't factor into the test?

Reply:
Zero-ing out the file is the same as writing a blank file to this location. The idea was that this test was for a fillable cache, which needed to start from 0 at the beginning of each test. I'll add some comments.